docs/user/troubleshooting/evnt-01-eventing-troubleshooting.md

```diff
@@ -16,21 +16,21 @@ Trouble with Kyma Eventing may be caused by various issues, so this document gui
 
 Follow these steps to detect the source of the problem:
 
-### Step 1: Check the status of the EventingBackend CR
+### Step 1: Check the status of the Eventing CR
 
-1. Check the EventingBackend CR. Is the field **EVENTINGREADY** `true`?
+1. Check the Eventing CR. Is the field **State** `Ready`?
 
     ```bash
-    kubectl -n kyma-system get eventingbackends.eventing.kyma-project.io
+    kubectl -n kyma-system get eventings.operator.kyma-project.io
     ```
 
-2. If **EVENTINGREADY** is `false`, check the exact reason of the error in the status of the CR by running the command:
+2. If **State** is not `Ready`, check the exact reason of the error in the status of the CR by running the command:
```

Comment:
Just want to make sure: shouldn't it be `Not Ready`/ `NotReady`/ `Not ready` instead of "not `Ready`"?

Reply:
There is no `NotReady` state. The possible values for `state` are `Ready`, `Error`, `Warning` and `Deletion`.